### PR TITLE
FAQ Application

### PR DESCRIPTION
**Pull request by khushi-purwar**
#119 solved.  I have added the FAQ Accordion. Please check my PR @Ayushparikh-code 

ss1

### CODE REVIEW

File: FAQ Application/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Description
+
+
+## Stacks Used
+* JavaScript
+* HTML & CSS  
+
+## ScreenShot
+
```
Comment (Ayushparikh-code) [inline FAQ Application/README.md:11]: add screenshot here

**Review by Ayushparikh-code — CHANGES_REQUESTED** [review]
screenshot not showing

File: FAQ Application/README.md (hunk 1 above)
Comment (khushi-purwar) [reply FAQ Application/README.md:11]: done!

File: FAQ Application/style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+  }
+  
+  @media screen and (max-width: 1024px) {
+    section {
+      margin: 0 auto;
+    }
+  }
+
+
```
Comment (chicken-biryani) [inline FAQ Application/style.css:101]: Add a new line

File: FAQ Application/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# FAQ Accordion
+
+## Description
+
+
```
Comment (chicken-biryani) [inline FAQ Application/README.md:5]: The description is blank. Also, add a "Use of this project" section and describe where can this project be used.

**Review by chicken-biryani — COMMENTED** [review]
Also add your project link and description to index.md

**Review by Ayushparikh-code — APPROVED** [review]
LGTM

**Pull request merged**